apisix/core/config_etcd.lua

```diff
@@ -74,6 +74,7 @@ if not is_http then
 end
 local created_obj  = {}
 local loaded_configuration = {}
+local configuration_loaded_time
```

Comment:
I can’t sure if we can use Lua global var to cache the value. If I do this, will store it in sharedict 
Pls add test it to confirm

Reply:
Okay, I will add a test case, but this one is obviously working. You can see another variable above `loaded_configuration`, which uses the same mechanism to allow worker processes to reuse the etcd data loaded when the master process starts.